src/content-helper/editor-sidebar/performance-details/component.tsx

```diff
@@ -73,233 +221,82 @@ export function PerformanceDetails(
-): JSX.Element {
-	const data = props.data;
-
-	// Remove unneeded totals to simplify upcoming map() calls.
-	delete data.referrers.types[ 'totals' as unknown as number ];
-
-	// Returns an internationalized referrer title based on the passed key.
-	const getKeyTitle = ( key: string ): string => {
-		switch ( key ) {
-			case 'social': return __( 'Social', 'wp-parsely' );
-			case 'search': return __( 'Search', 'wp-parsely' );
-			case 'other': return __( 'Other', 'wp-parsely' );
-			case 'internal': return __( 'Internal', 'wp-parsely' );
-			case 'direct': return __( 'Direct', 'wp-parsely' );
-		}
-
-		return key;
-	};
-
-	return (
-		<div className="section referrer-types">
-			<div className="section-title">{ __( 'Referrers (Page Views)', 'wp-parsely' ) }</div>
-
-			<div className="multi-percentage-bar">{
-				Object.entries( data.referrers.types ).map( ( [ key, value ] ) => {
-					const ariaLabel = sprintf(
-						/* translators: 1: Referrer type, 2: Percentage value, %%: Escaped percent sign */
-						__( '%1$s: %2$s%%', 'wp-parsely' ),
-						getKeyTitle( key ), value.viewsPercentage
-					);
-
-					return (
-						<div aria-label={ ariaLabel }
-							className={ 'bar-fill ' + key } key={ key }
-							style={ { width: value.viewsPercentage + '%' } }>
-						</div>
-					);
-				} ) }
-			</div>
-
-			<table>
-				<thead>
-					<tr>{
-						Object.keys( data.referrers.types ).map( ( key ) => {
-							return <th key={ key }>{ getKeyTitle( key ) }</th>;
-						} ) }
-					</tr>
-				</thead>
-				<tbody>
-					<tr>{
-						Object.entries( data.referrers.types ).map( ( [ key, value ] ) => {
-							return <td key={ key }>{ formatToImpreciseNumber( value.views ) }</td>;
-						} ) }
-					</tr>
-				</tbody>
-			</table>
-		</div>
-	);
-}
-
-/**
- * Outputs the "Top Referrers" section.
- *
- * @param {PerformanceSectionProps} props The props needed to populate the section.
- */
-function TopReferrersSection(
-	props: Readonly<PerformanceSectionProps>
-): JSX.Element {
-	const data = props.data;
-	let totalViewsPercentage = 0;
-
-	return (
-		<div className="section top-referrers">
-			<table>
-				<thead>
-					<tr>
-						<th scope="col">{ __( 'Top Referrers', 'wp-parsely' ) }</th>
-						<th colSpan={ 2 } scope="colgroup">{ __( 'Page Views', 'wp-parsely' ) }</th>
-					</tr>
-				</thead>
-				<tbody>{
-					Object.entries( data.referrers.top ).map( ( [ key, value ] ) => {
-						if ( key === 'totals' ) {
-							totalViewsPercentage = Number( value.viewsPercentage );
-							return null;
+				{ window.wpParselyPostUrl && (
+					<Button
+						className="wp-parsely-view-post"
+						variant={ 'primary' }
+						onClick={ () => {
+							Telemetry.trackEvent( 'editor_sidebar_view_post_pressed' );
+						} }
+						href={ window.wpParselyPostUrl }
+						rel="noopener"
+						target="_blank"
+					>
+						{
+							/* translators: %s: Post type */
+							sprintf( __( 'View this %s in Parse.ly', 'wp-parsely' ), 'post' )
```

Comment:
Interesting point. I think that for simplicity, we can keep "View this in Parse.ly". Looking forward to hear more input on this :)

Reply:
Agree. "View this in Parse.ly" is great.

@acicovic do you mean inverse the "View this in Parse.ly" button and the period filter so that the filter is below the button?

Reply:
@danielabloch, yes. As it is right now, it's like the filter is related to the button, but this isn't the case. The filter is related to the information below, so moving it closer towards there makes more sense IMO. Augmenting the distance between them should also help.

Reply:
Initial thoughts: fine to keep remove the name of the post type for now. We should figure out how to properly truncate that if needed. The dynamic names of database stuff has always been a Wordpress feature I've appreciated, so it would be good to keep that convention here instead of just calling it "this." But fine if it needs more work to design around the technical constraints. 

As for the button, I would definitely put it above the period select. I'll need to modify the design if we're intent on moving the view in parse.ly button inside performance so the spacing can get just right.

Reply:
Just updated Figma to take the button out of the header and put it at the bottom of performance, From a UX/markup perspective I think that's what makes the most sense.

Reply:
@dabowman, @vaurdan, Figma seems broken for me right now, but this separation makes sense to me.

Reply:
I think this is how it was before as well.

Reply:
@dabowman thank you for you input! That makes perfect sense, and looks way better IMO. I will apply that to the PR.

Reply:
I moved this to the bottom of the performance tab.

![image](https://github.com/Parsely/wp-parsely/assets/5804923/57f5e97f-2a77-4e37-a456-a8b741958f65)